src/Symfony/Component/Mime/Crypto/DkimSigner.php

```diff
@@ -57,7 +57,7 @@ public function __construct(string $pk, string $domainName, string $selector, ar
             'body_show_length' => false,
             'header_canon' => self::CANON_RELAXED,
             'body_canon' => self::CANON_RELAXED,
-            'headers_to_ignore' => [],
+            'headers_to_ignore' => ['X-Transport'],
```

Comment:
As we set the default options here, this can be overridden. As we always want to remove it, I would force it like done for `return-path` after line 70 below.

Reply:
Done.

Is it correct, that we don't provide for users possibility to override this option and not to ignore X-Transport? For example if they use `symfony/mime`'s DKIM Signer without `symfony/mailer`? Probably there are no such users though. And DKIM Signer from `symfony/mime` is supposed to use only with `symfony/mailer`, right?

Reply:
Your patch is good enough IMHO.